### PR TITLE
feat(STONEINTG-891): add test_metadata task and pipeline

### PR DESCRIPTION
**Pull request by dirgim**
* This example shows how the test metadata can be derived from the SNAPSHOT parameter as well as the PaC and Konflux/appstudio labels and annotations

### CODE REVIEW

File: tasks/test_metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        - name: TARGET_GIT_URL
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.annotations['pac.test.appstudio.openshift.io/repo-url']
+        - name: TARGET_GIT_BRANCH
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.annotations['pac.test.appstudio.openshift.io/branch']
+      script: |
+        #!/bin/sh
```
Comment (flacatus) [inline tasks/test_metadata.yaml:60]: Can we get also the Author of the commit?

Comment (dirgim) [reply tasks/test_metadata.yaml:60]: Yup, that should be available as standard afaik.

Comment (dirgim) [reply tasks/test_metadata.yaml:60]: Added the `sender` result, let me know if that works for you.

**Review by jsztuka — APPROVED** [review]
lgtm

**Review by hongweiliu17 — APPROVED** [review]

**Pull request merged**